aem/core_aem/src/main/java/com/adobe/aio/aem/workspace/ocd/WorkspaceConfig.java

```diff
@@ -26,6 +26,10 @@
       description = "Comma separated list of metascopes associated with your API (`/s/event_receiver_api,/s/ent_adobeio_sdk` for instance) (project.workspace.details.credentials.jwt.meta_scopes)")
   String aio_meta_scopes() default "/s/ent_adobeio_sdk";
 
+  @AttributeDefinition(name = "OAuth Scopes",
+          description = "Comma separated list of oauth scopes associated with your API (project.workspace.details.credentials.oauth_server_to_server.scopes)")
```

Comment:
could you add a statement here in the `description` to inform this `
  String `  attribute is to be used for oauth only.
and similarly on line 25 could you upadte `Meta Scopes` to `JWT Meta Scopes`  
as well as the `description` to inform this `aio_meta_scopes` attribute is to be used for JWT only.

Reply:
could you also update https://github.com/adobe/aio-lib-java/tree/main/aem/core_aem#workspace-configuration accordingly ?

Reply:
Review Comments Incorporated